front/app/containers/Admin/projects/project/permissions/components/PhaseAccordion/messages.ts

```diff
@@ -0,0 +1,8 @@
+import { defineMessages } from 'react-intl';
+
+export default defineMessages({
+  phase: {
+    id: 'app.containers.AdminPage.groups.permissions.phase',
+    defaultMessage: 'Phase ',
```

Comment:
Is it wise to define the message with a space at the end? Could get mistranslated without a space?

Reply:
It's quite common so I think it's OK. I think crowdin also gives you a warning if the original copy ends with a space and you accidentally remove it